demisto_sdk/commands/lint/linter.py

```diff
@@ -720,7 +720,7 @@ def _docker_run_linter(self, linter: str, test_image: str, keep_container: bool)
         exit_code = SUCCESS
         output = ""
         try:
-            container: docker.models.containers.Container = Docker.create_container(
+            container: docker.models.containers.Container = get_docker_helper().create_container(
```

Comment:
wdyt of `get_docker_function` for a name? I'm not a fan of `_helper()`s

Reply:
get_docker seems nicer than get_docker_function. It also starts to lose some meaning though